resources/lang/en/common.php

```diff
@@ -12,7 +12,8 @@
  */
 
 return [
-    'abbrev-days'          => 'D ',
+    'a-an-art' => '',
```

Comment:
What is this for?

Reply:
For switch between **a/an** (article word in English). In code `@lang(trans_choice('common.a-an-art',false))` represents "a" and true for "an".

Reply:
Perfect thanks!